cli.go

```diff
@@ -208,5 +211,23 @@ func (app *TGFApplication) Run() int {
 		Printf("tgf v%s\n", version)
 		return 0
 	}
+
+	if !app.NoUpdate {
+		didUpdate := RunUpdate()
+
+		if didUpdate {
+			Println("tgf updated !")
+			// Proceed, forwarding arguments
+			cmd := exec.Command(os.Args[0], os.Args[1:]...)
```

Comment:
This code should also be in the `_linux` file. This is not what will happen on Windows probably